src/vos/tests/vts_io.c

```diff
@@ -1622,7 +1649,7 @@ io_simple_punch(void **state)
 	struct io_test_args	*arg = *state;
 	int			rc;
 
-	arg->ta_flags = TF_PUNCH;
+	arg->ta_flags = TF_PUNCH | TF_REC_EXT;
```

Comment:
Why this test is changed from punch SV to punch EV?

Reply:
Because it punches the last key and the last key (from a previous test) happens to be an EVT.

These tests need some major rework, IMO to test each thing in isolation rather than relying on state from previous tests to setup the proper state but I didn't want to tackle that here.

Reply:
I see, thanks.